stablehlo/reference/InterpreterOps.td

```diff
@@ -74,4 +74,38 @@ def Interpreter_RunParallelOp : Op<Interpreter_Dialect, "run_parallel", []> {
   let hasVerifier = 1;
 }
 
+def Interpreter_ProbeOp : Op<Interpreter_Dialect, "probe",
+    [SameOperandsAndResultShape, SameTypeOperands]> {
```

Comment:
Is `SameTypeOperands` needed with one input?

Reply:
Good catch, I don't think so.